main.go

```diff
@@ -56,7 +57,7 @@ func main() {
 		fmt.Fprintf(os.Stderr, "cannot find binary %q in any of:\n", binary)
 		workspaces := filepath.SplitList(build.Default.GOPATH)
 		for i, workspace := range workspaces {
-			path := filepath.Join(workspace, "bin", binary)
+			path := filepath.Join(workspace, "bin", binaryName(binary))
```

Comment:
You missed another place where `binary` is used above, on line 57:

```Go
fmt.Fprintf(os.Stderr, "cannot find binary %q in any of:\n", binary)
```

It's better to just declare a local variable `binary` once and reuse it. E.g.:

```Go
// Print output.
for command, matched := range filter {
	binary := binaryName(command)
	...
```